frontend/src/components/SideBar.tsx

```diff
@@ -8,169 +8,88 @@
 // or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
 // OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
 // limitations under the License.
+import {SideNavigation, SideNavigationProps} from '@awsui/components-react'
 import * as React from 'react'
-import {Link, useLocation} from 'react-router-dom'
-import {setState, useState, isGuest, isUser, isAdmin} from '../store'
-
-// Icons
-import GridOnIcon from '@mui/icons-material/GridOn'
-import ImageIcon from '@mui/icons-material/Image'
-import LayersIcon from '@mui/icons-material/Layers'
-import HomeIcon from '@mui/icons-material/Home'
-import GroupIcon from '@mui/icons-material/Group'
 import {useTranslation} from 'react-i18next'
-import Image from 'next/image'
+import {useLocation, useNavigate} from 'react-router-dom'
+import {USER_ROLES_CLAIM} from '../auth/constants'
+import {isAdmin, isGuest, isUser, useState} from '../store'
 
-export function SideBarIcons() {
-  const drawerOpen = useState(['app', 'sidebar', 'drawerOpen'])
+const MATCH_UP_UNTIL_SECOND_FORWARD_SLASH = /\/[^\/]+/
 
+export default function SideBar() {
+  const {t} = useTranslation()
+  const navigate = useNavigate()
   const location = useLocation()
-  let defaultPage = isGuest() ? 'home' : 'clusters'
-  let section = location && location.pathname && location.pathname.substring(1)
-  section =
-    section &&
-    (section.indexOf('/') !== -1
-      ? section.substring(0, section.indexOf('/'))
-      : section)
-  section = section || defaultPage
+  const defaultPage = isGuest() ? '/home' : '/clusters'
+  const currentHref = MATCH_UP_UNTIL_SECOND_FORWARD_SLASH.exec(
```

Comment:
suggestion(non-blocking): it can be simplified with `location.pathname.split('/')[0])`. I think it's a bit more idiomatic for JS and doesn't need a proper naming choice